consul-secure-on-kubernetes/provision.courseData.sh

```diff
@@ -0,0 +1,70 @@
+cat << 'EOFSRSLY' > /tmp/provision.sh
```

Comment:
```suggestion
cat << 'EOF' > /tmp/provision.sh
```

I like humor as much as the next person, but for public code let's stick to non-silly variable names like `EOF`